### ISSUE TITLE
Building error with variable has incomplete type 'struct bxf_spawn_params'

### ISSUE THREAD

**Issue by toonaive0618** [Snaipe/Criterion#235]
I have already installed the Boxfort, but it still reports this error when building, please check this issue 
and sorry for the disturb

**Comment by Snaipe** [comment]
Did you manage to solve your issue?

**Comment by toonaive0618** [comment]
yep, thanks for your reply, If I manually install the Boxfort, then install Criterion, it will report this error, if It auto installed by Criterion, it works, didn't report any error. So I closed this issue,

**Comment by Sapein** [comment]
I should note that I hat this issue today as well.

The issue is caused due to a version difference between Boxfort in what Criterion uses and what the current version of Boxfort has.

You see Criterion uses Boxfort with the head at https://github.com/diacritic/BoxFort/commit/7ed0cf2120926935bfd1b24e3fdfd63d70b1999c whereas the latest head is at https://github.com/diacritic/BoxFort/commit/c99f5279bba41b2dcf1826daa5daa23d08b0b5ee

in that commit bxf_spawn_params was renamed to bxf_spawn_param_s. So this issue only arises if someone compiles and installs BoxFort from the latest master of the git repo, but when Criterion builds it, it pulls in a version from before that change. So either Criterion should be updated to reference the changes to BoxFort, or the documentation should be updated to reference that you need a commit from _before_ https://github.com/diacritic/BoxFort/commit/c99f5279bba41b2dcf1826daa5daa23d08b0b5ee
